macros.xml

```diff
@@ -2,6 +2,7 @@
 <macros>
     <xml name="requirements">
         <requirements>
+            <container type="docker">labsyspharm/ashlar:1.13.0</container>
```

Comment:
Can the VERSION token be used for the docker tag?

Reply:
Will be cool